.github/workflows/build_wheels.yml

```diff
@@ -1,4 +1,4 @@
-name: run-tests-local
+name: build-wheels
```

Comment:
Seems like this was a mistake?

Reply:
We should still be testing the wheel builds.

Reply:
Another thought here, 

we should add
```bash
export SMARTSIM_SUFFIX=develop
```
prior to the wheel build so the wheels are named with the dirty version.

Reply:
We are still testing the wheel builds. Wheel builds occur in `build_wheels.yml` and the extended build matrix which will eventually contain code coverage reporting is in `run_tests.yml`.

I added 
```yml
env:
    SMARTSIM_SUFFIX: "develop"
```
to the wheel builds in my most recent push.